packages/react-server-dom-parcel/src/shared/ReactFlightClientConfigBundlerParcel.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Thenable} from 'shared/ReactTypes';
+
+import type {ImportMetadata} from './ReactFlightImportMetadata';
+
+import {ID, NAME, BUNDLES} from './ReactFlightImportMetadata';
+
+export type ServerManifest = {
+  [string]: Array<string>,
+};
+export type SSRModuleMap = null;
+export type ModuleLoading = null;
+export type ServerConsumerModuleMap = null;
+export type ServerReferenceId = string;
+
+export opaque type ClientReferenceMetadata = ImportMetadata;
+
+// eslint-disable-next-line no-unused-vars
+export opaque type ClientReference<T> = {
+  // Module id.
+  id: string,
+  // Export name.
+  name: string,
+  // List of bundle URLs, relative to the distDir.
+  bundles: Array<string>,
+};
+
+export function prepareDestinationForModule(
+  moduleLoading: ModuleLoading,
+  nonce: ?string,
+  metadata: ClientReferenceMetadata,
+) {
+  return;
+}
+
+export function resolveClientReference<T>(
+  bundlerConfig: null,
+  metadata: ClientReferenceMetadata,
+): ClientReference<T> {
+  // Reference is already resolved during the build.
+  return {
+    id: metadata[ID],
+    name: metadata[NAME],
+    bundles: metadata[BUNDLES],
+  };
```

Comment:
Is there a reason this isn't just an identity function returning the same type of object?